### PR TITLE
Add license info to Microsoft extensions

### PR DESCRIPTION
**Pull request by bghgary**
See #2210.

TODO
- [x] Adobe signs off on EXT_lights_image_based wording.

### CODE REVIEW

**Review by MiiBond — APPROVED** [review]

File: extensions/2.0/Vendor/EXT_lights_image_based/README.md [hunk 1]
```diff
@@ -126,3 +126,5 @@ Each scene can have a single IBL light attached to it by defining the `extension
 | `specularImages` | Declares an array of the first N mips of the prefiltered cubemap. Each mip is, in turn, defined with an array of 6 images, one for each cube face. i.e. this is an Nx6 array. | :white_check_mark: Yes |
 | `specularImageSize` | The dimension (in pixels) of the first specular mip. This is needed to determine, pre-load, the total number of mips needed. | :white_check_mark: Yes |
 
+## License
+Adobe and Microsoft has made this Specification available under the Open Web Foundation Agreement Version 1.0, which is available at https://www.openwebfoundation.org/the-agreements/the-owf-1-0-agreements-granted-claims/owfa-1-0.
```
Comment (lexaknyazev) [inline extensions/2.0/Vendor/EXT_lights_image_based/README.md:130]: has -> have

**Review by lexaknyazev — COMMENTED** [review]
Also please add a line break after `## License` in all 5 files.

File: extensions/2.0/Vendor/EXT_lights_image_based/README.md (hunk 1 above)
Comment (bghgary) [reply extensions/2.0/Vendor/EXT_lights_image_based/README.md:130]:
```suggestion
Adobe and Microsoft have made this Specification available under the Open Web Foundation Agreement Version 1.0, which is available at https://www.openwebfoundation.org/the-agreements/the-owf-1-0-agreements-granted-claims/owfa-1-0.
```

**Comment by bghgary** [comment]
@lexaknyazev I addressed your comments.

**Review by lexaknyazev — APPROVED** [review]

**Pull request merged**